inspirehep/modules/migrator/tasks.py

```diff
@@ -378,8 +337,11 @@ def create_record(record, force=True, dry_run=False):
     else:
         json = hep.do(record)
 
-    if dry_run:
-        return errors, json
+    if '$schema' in json:
+        json['$schema'] = url_for(
+            'invenio_jsonschemas.get_schema',
+            schema_path="records/{0}".format(json['$schema'])
+        )
```

Comment:
@mihaibivol, where's the other place where you were doing something similar? Is this solution better?